frontend/public/components/resource-quota.jsx

```diff
@@ -35,64 +55,125 @@ const Row = ({obj: rq}) => <div className="row co-resource-list__item">
-  const used = _.get(quota, ['status', 'used', resourceType]);
-  const proportionUsed = getProportionUsed(used, max);
+  const proportions = getProportions(quota, resourceType);
   return <div className="row co-m-row">
     <div className="col-sm-4 col-xs-6 co-break-word">{resourceType}</div>
-    <div className="col-sm-2 hidden-xs co-resource-quota-icon"><ProportionIcon proportion={proportionUsed} /></div>
-    <div className="col-sm-3 col-xs-3">{used}</div>
-    <div className="col-sm-3 col-xs-3">{max}</div>
+    <div className="col-sm-2 hidden-xs co-resource-quota-icon"><ProportionIcon percent={proportions.percentUsed} /></div>
+    <div className="col-sm-3 col-xs-3">{proportions.used}</div>
+    <div className="col-sm-3 col-xs-3">{proportions.max}</div>
   </div>;
 };
 
-const Details = ({obj: rq}) => <React.Fragment>
-  <div className="co-m-pane__body">
-    <SectionHeading text="Resource Quota Overview" />
-    <ResourceSummary resource={rq} podSelector="spec.podSelector" showNodeSelector={false} />
-  </div>
-  <div className="co-m-pane__body">
-    <SectionHeading text="Resource Usage" />
-    <div className="co-m-table-grid co-m-table-grid--bordered">
-      <div className="row co-m-table-grid__head">
-        <div className="col-sm-4 col-xs-6">Resource Type</div>
-        <div className="col-sm-2 hidden-xs">Capacity</div>
-        <div className="col-sm-3 col-xs-3">Used</div>
-        <div className="col-sm-3 col-xs-3">Max</div>
+const QuotaScopes = ({quota}) => {
+  const scopes = _.get(quota, ['spec', 'scopes']);
+  if (!scopes) {
+    return <dd>-</dd>;
+  }
+  const retVal = scopes.map(scope => {
+    const scopeObj = _.get(quotaScopes, scope);
+    return scopeObj ?
+      <dd key={scope}>
+        <div className="co-resource-quota-scope__label">{scopeObj.label}</div>
+        <div className="co-resource-quota-scope__description">{scopeObj.description}</div>
+      </dd>
+      : <dd key={scope} className="co-resource-quota-scope__label">{scope}</dd>;
+  });
+  return retVal;
+};
+
+const showCharts = resourceTypes => {
+  const chartResourceTypes = ['requests.cpu', 'limits.cpu', 'requests.memory', 'limits.memory'];
+  return _.intersection(resourceTypes, chartResourceTypes).length > 0;
+};
+
+const Details = ({obj: rq}) => {
+  const resourceTypes = getResourceTypes(rq);
+  const showChartRow = showCharts(resourceTypes);
+  const popover = (
+    <Popover>
+      Requests are the amount of resources you expect to use. These are used when establishing if the cluster can fulfill your Request.<br />
+      <br />
+      Limits are a maximum amount of a resource you can consume. Applications consuming more than the Limit may be terminated.<br />
+      <br />
+      A cluster administrator can establish limits on both the amount you can Request and your Limits with a &nbsp;
+      <a href={helpLink('dev_guide/compute_resources.html#dev-compute-resources')} target="_blank" rel="noopener">Resource Quota</a>.
```

Comment:
```suggestion
      <a href={helpLink('dev_guide/compute_resources.html#dev-compute-resources')} target="_blank" rel="noopener noreferrer">Resource Quota</a>.
```

Reply:
Add a value to the help topics enum in documentation.tsx instead of passing the path here.